src/main/resources/config-test-test.xml

```diff
@@ -40,13 +41,15 @@
   <!-- ### actions to take on metadata formats (in order of preference) ### -->
   <actions>
     <format match="namespace" value="http://www.clarin.eu/cmd/1">
+      <action type="transform" file="https://raw.githubusercontent.com/menzowindhouwer/oai-harvest-manager/develop/src/main/resources/filter.xsl"/>
```

Comment:
Maybe it's a bit brittle to tie this to a file in the develop branch. Maybe best to use a tag? Though that will probably be easier to do after the release though..

Reply:
Yes this one will change after the merge and the release. If we use it in production it should move to the config repository.